README.md

```diff
@@ -303,32 +306,34 @@ Meters report consumption over the service.
+`meter_elec`    | energy, power, reactive power, frequency, voltage, current, power_factor, pulses | Electric meters
+`meter_gas`     | volume, energy, pulses, flow, pressure                                           | Gas meter
+`meter_water`   | volume, energy, pulses, flow, pressure                                           | Water meter
+`meter_heating` | energy consumption                                                               | Heating meter
+`meter_cooling` | energy consumption                                                               | Cooling meter
 
 #### Interfaces
 
-Type | Interface                     | Value type | Properties                         | Description
------|-------------------------------|------------|------------------------------------|-------------
-in   | cmd.meter.get_report          | string     | direction                          | Value - is a unit. May not be supported by all meters.
-in   | cmd.meter.reset               | null       |                                    | Resets all historical readings.
-out  | evt.meter.report              | float      | unit, prv_data, delta_t, direction |
-out  | evt.meter_ext.report          | float_map  |                                    | [Extended meter report](#extended-report-object) with up to 17 data points
-in   | cmd.meter_ext.get_report      | null       |                                    | Request extended report
+Dir  | `type`                        | `val_t`            | `prop`                             | Description
+-----|-------------------------------|--------------------|------------------------------------|-------------
+in   | cmd.meter.get_report          | string             | direction                          | Triggers evt.meter.report. An argument is "export" or "import".
+in   | cmd.meter.reset               | null               |                                    | Resets all historical readings.
+out  | evt.meter.report              | float              | unit, prv_data, delta_t, direction | Meter reading
+out  | evt.meter_ext.report          | float_map          |                                    | [Extended meter report](#extended-report-object) with up to 17 data points
+in   | cmd.meter_ext.get_report      | null               |                                    | Request extended report
 
 #### Interface props
 
-Name         | Value example | Description
--------------|---------------|-------------
-`delta_t`    |               | Elapsed time in seconds between the 'Meter Value' and the 'Previous Meter Value' measurements.
-`prv_data`   |               | Previous meter reading.
-`unit`       | "kWh"         | One of sup_units. For meter_unknown it will be a number.
-`direction`  | "export"      | Defines whether reading is based on consumption (import) or production (export). Applicable for every meter except for meter_elec.
+Name         | Value type                  | Description
```

Comment:
Another PR for that kind of changes, please :)
It seems like "Value type" as You described it, is something like a description of an interface props(?)
So it's exactly the same as if You had it all in "Description" column(?)

First row could be a nice example:
You propose:
`delta_t` | int [seconds] | Time between the current and the previous meter reading.

instead of
`delta_t` | 60 | Elapsed time in seconds between the 'Meter Value' and the 'Previous Meter Value' measurements.

Reply:
exemplary json can be added belowe. Feel free to do it.